HERRAMIENTA: "eiiichecker"
	FECHA: "2018-12-10 13:45:58.944419"
	PAGINA WEB: "89"
	ERROR: "Exception('No se ha podido realizar la evaluaci√≥n',)"

HERRAMIENTA: "observatorio"
	FECHA: "2018-12-10 13:47:00.054745"
	PAGINA WEB: "89"
	ERROR: "ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='observatorioweb.ups.edu.ec', port=80): Read timed out. (read timeout=60)",),)"

HERRAMIENTA: "wave"
	FECHA: "2018-12-10 14:01:59.151742"
	PAGINA WEB: "222"
	ERROR: "ConnectTimeout(MaxRetryError("HTTPConnectionPool(host='wave.webaim.org', port=80): Max retries exceeded with url: /api/request?key=****&url=http://www.aemet.es/en/portada&format=json&reporttype=2 (Caused by ConnectTimeoutError(<requests.packages.urllib3.connection.HTTPConnection object at 0x7f61a0c80ef0>, 'Connection to wave.webaim.org timed out. (connect timeout=15)'))",),)"

HERRAMIENTA: "observatorio"
	FECHA: "2018-12-10 14:03:58.264743"
	PAGINA WEB: "224"
	ERROR: "ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='observatorioweb.ups.edu.ec', port=80): Read timed out. (read timeout=60)",),)"

